Classes/Issues/Comments/Markdown/String+DetectShortlink.swift

```diff
@@ -59,6 +59,14 @@ extension String {
                     )
                     ])
                 .restore()
+            
+            //if match's trailing character is not a number, include that
+            if let right = substring(with: NSRange(location: match.range.right - 1, length: 1)) {
+                let nums: Set<Character> = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"]
+                if Set(right).isSubset(of: nums) == false {
+                    builder.add(text: right)
+                }
+            }
```

Comment:
not sure why I wrote this elaborate number checking scheme instead of just seeing if the string could be cast to an Int